src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -527,7 +530,21 @@ def build(self, notification_uuid: str | None = None) -> SlackBlock:
         # build title block
         title_text = f"<{title_link}|*{escape_slack_text(title)}*>"
 
-        if self.group.issue_category == GroupCategory.ERROR:
+        # build up actions text
+        if self.actions and self.identity and not action_text:
+            action_text = get_action_text(text, self.actions, self.identity)
```

Comment:
we have this originally because we don't want to rebuild the message buttons when somebody takes an action (see https://github.com/getsentry/sentry/pull/64431). so i have to check inside here and say that yes, somebody is taking an action

Reply:
Can we pull out all the title emoji logic? a method that would return the right title emoji in a block to append getting group and actions, or none